test/grammar/builtins/net/cidr/main.k

```diff
@@ -5,3 +5,4 @@ cidr1 = net.hosts_in_CIDR("192.168.1.0/24")
 cidr2 = net.subnets_from_CIDR("192.168.1.0/24")
 cidr3 = net.is_IP_in_CIDR("192.168.1.1", "192.168.1.0/24")
 cidr4 = net.is_IP_in_CIDR("192.168.0.1", "192.168.1.0/24")
+cidr5 = net.is_IP_in_CIDR('2001:db8::1', "2001:db8::/64")
```

Comment:
There should be a LOT more tests:
* First in CIDR
* Last in CIDR
* One before first in CIDR
* One after last in CIDR
* `0.0.0.0/0` and `::/0`
* IPv4 in `::/0`
* IPv4 in IPv6 CIDR and vice versa
* Invalid CIDRs
* Invalid IPs
* IPv4 /32 and IPv6 /128